hardhat.config.ts

```diff
@@ -43,6 +43,10 @@ const config: HardhatUserConfig = {
   networks: {
     hardhat: {
       chainId: 1337,
+      forking: {
```

Comment:
moved to a forking test suite because we need to test on real erc20 addresses to test the receipt renderer `humanRewardAmountAndSymbol` function.

Reply:
Don't we need to run a different command when we wanna test fork mainnet? Or does running `yarn test` automatically apply the mainnet fork?

Reply:
`yarn test` will now always apply the mainnet fork, yes.